packages/indexer-agent/src/commands/start.ts

```diff
@@ -359,6 +379,36 @@ export async function createNetworkSpecification(
   const chainId = await fetchChainId(networkProvider.url)
   const networkIdentifier = resolveChainId(chainId)
 
+  // Warn about inappropriate max block distance for subgraph threshold checks for given networks.
```

Comment:
why are the warnings for Arbitrum only?

Reply:
Default values were tested and set for Ethereum networks (Mainnet & Goerli), but Arbitrum has a faster block speed, so the default config will most likely stall the Agent if used on Arbitrum networks.